src/components/navigation/Link.tsx

```diff
@@ -23,35 +28,45 @@ const Link = function Link({
   elementRef,
 
   underline = 'none',
-  color = 'brand',
+  color,
+  unstyled = false,
+  variant = 'brand',
 
   ...htmlAttributes
 }: LinkProps) {
+  // Map from deprecated `color` prop to `variant` if `color` is present
+  const theme = typeof color === 'string' ? color : variant;
+  const styled = !unstyled;
+  const themed = styled && theme !== 'custom';
+
   return (
-    <LinkBase
+    <a
       data-component="Link"
+      rel="noopener noreferrer"
       {...htmlAttributes}
-      classes={classnames(
-        'rounded-sm',
-        // NB: Base classes are applied by LinkBase
```

Comment:
Not anymore. This is a good thing because even this small abstraction caused a regression in one of these components recently.